projects/plugins/jetpack/class.json-api.php

```diff
@@ -1021,26 +1021,26 @@ public function wp_count_comments( $post_id ) {
 		);
 
 		/**
-		 * Exclude certain comment types from comment counts in the REST API.
+		 * Include certain comment types in comment counts in the REST API.
 		 *
-		 * @since 6.9.0
+		 * @since 10.9.x
 		 * @module json-api
 		 *
-		 * @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
+		 * @param array Array of comment types to include (default: 'comment', 'pingback', 'trackback')
 		 */
-		$exclude = apply_filters(
-			'jetpack_api_exclude_comment_types_count',
-			array( 'order_note', 'webhook_delivery', 'review', 'action_log' )
+		$include = apply_filters(
```

Comment:
For a deprecation, I think it'd be best to leave around the existing filter/function at least for a period of time. You could switch the existing `apply_filters` call to [apply_filters_deprecated](https://developer.wordpress.org/reference/functions/apply_filters_deprecated/). If you search for `apply_filters_deprecated` in the Jetpack monorepo there are a couple examples.